src/tracing/exporter.js

```diff
@@ -34,7 +34,7 @@ export class SpanExporter {
       return { resourceSpans: [] };
     }
 
-    const resource = options.resource || (spans[0] && spans[0].resource) || {};
+    const resource = (spans[0] && spans[0].resource) || {};
```

Comment:
Could you update the jsdoc, too? In fact, I think we could remove the options param altogether.